app.rb

```diff
@@ -0,0 +1,172 @@
+    puts '7 - Exit'
+    @number = gets
+    until @number.to_i.between?(1, 7)
+      puts 'Please write a number between 1 and 7'
+      @number = gets
+    end
+  end
+
+  def options
+    case @number.to_i
+    when 1
+      list_all_books
+      run
+    when 2
+      list_all_people
+      run
+    when 3
+      create_person
+      run
+    when 4
+      create_book
+      run
+    when 5
+      create_rental
+      run
+    when 6
+      list_all_rentals_for_id
+      run
+    end
+  end
+
+  def run
+    show_menu
+    if @number.to_i == 7
+      exit
+    else
+      options
+    end
+  end
+
+  def create_person
+    print 'Please choose 1 for create a student or 2 for create a teacher: '
+    teacher_or_student = gets.chomp.to_i
+    case teacher_or_student
+    when 1
+      create_student
+    when 2
+      create_teacher
+    else
+      puts 'Invalid option, try again'
+      create_person
+    end
+  end
+
+  def create_student
+    print 'Age?: '
+    age = gets.chomp.to_i
+    print 'Name?: '
+    name = gets.chomp
+    print 'Has parent permission?[Y/N]: '
+    parent_permission = gets.chomp
+    case parent_permission
+    when 'y' || 'Y'
+      new_student = Student.new(age, name, true)
+      @people << new_student
+      puts 'Person created sucessfully'
+    when 'n' || 'N'
+      new_student = Student.new(age, name, false)
+      @people << new_student
+      puts 'Person created sucessfully'
+    else
+      puts 'Invalid, please try again'
+      create_student
+    end
+  end
+
+  def create_teacher
+    print 'Age?: '
+    age = gets.chomp.to_i
+    print 'Name?: '
+    name = gets.chomp
+    print 'Specialization?: '
+    specialization = gets.chomp
+    new_teacher = Teacher.new(age, specialization, name)
+    @people << new_teacher
+    puts 'Person created sucessfully'
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
+    new_book = Book.new(title, author)
+    @books << new_book
+    puts "Book #{title} created succesfully"
+  end
+
+  def create_rental
+    if @books.empty? || @people.empty?
+      puts 'You need a book and a person to create a new renta, please add both first'
+    else
+      puts 'Select a book by title'
+      @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+      print 'Write the title of the book that you want: '
+      book_title = gets.chomp
+      book = @books.select { |b| b.title == book_title }
+      puts 'Select the person who is renting the book by the ID'
+      @people.each { |person| puts "Name: #{person.name}, Age: #{person.age}, id: #{person.id}" }
+      person_id = gets.chomp.to_i
+      person = @people.select { |per| per.id == person_id }
+      print 'Please enter a date in the next format yyyy-mm-dd: '
+      date = gets.chomp
+      new_rental = Rental.new(date, book[0], person[0])
+      @rentals << new_rental
+      puts 'Rental created succesfully'
```

Comment:
- [ ] The same issue as in creating person after creating a rental it didn't return the successful message. Please 🙏🏽 fix this issue for a better UX.

Reply:
Good day @franclobo I'm going to correct the menu. I noticed that the message is returning, as you can see even in the screenshot, what do you want me to correct?